Framework/PythonInterface/test/python/plugins/algorithms/IqtFitMultipleTest.py

```diff
@@ -123,7 +123,7 @@ def _validate_group_values(self, groupWS):
         # Check Calc
         calc = sub_ws.readY(1)
         self.assertEquals(round(calc[0], 5), 1)
-        self.assertEquals(round(calc[-1],7),0.0264651)
+        self.assertEquals(round(calc[-1],6),0.026465)
         # Check Diff
```

Comment:
@ElliotAOram : probably you know the meaning/implications of those changes well. It looks acceptable/sensible in principle?